src/dal_select2/static/autocomplete_light/select2.js

```diff
@@ -8,13 +8,24 @@
             var data_forward = {};
 
             for (var key in forward) {
+                var f = forward[key].split("->");
+                var src_name, dst_name;
+                if (f.length === 0) {
+                    continue;
+                } else if (f.length === 1) {
+                    src_name = f[0];
+                    dst_name = f[0];
+                } else {
+                    src_name = f.splice(0,1);
+                    dst_name = f.join("->");
+                }
```

Comment:
> Do you think it will work to have the script tag after the select tag ?

Yes, I have already implemented it in next changesets.

> we could also have a js callback type.
Actually, the content of script tag with custom `type` value is an arbitrary text which we parse as JSON when we are adding forward values (see new changes). If we need callbacks we can do some dirty tricks with `eval()` function, but it does not seem a good idea for me now.